packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.spec.js

```diff
@@ -1,91 +1,100 @@
-import { mount } from 'enzyme'
-import { DropButton, Grommet } from 'grommet'
+import { expect } from 'chai'
 import React from 'react'
-import zooTheme from '@zooniverse/grommet-theme'
+import sinon from 'sinon'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 import { task as mockTask } from '@plugins/tasks/survey/mock-data'
 import { default as Task } from '@plugins/tasks/survey'
 
 import FilterStatus from './FilterStatus'
-import Characteristics from '../Characteristics'
-import FilterButton from '../components/FilterButton'
 
 describe('Component > FilterStatus', function () {
-  let wrapper
+  this.timeout(0)
+
   let task = Task.TaskModel.create({
     characteristics: mockTask.characteristics,
     characteristicsOrder: mockTask.characteristicsOrder,
     images: mockTask.images,
     taskKey: 'T0',
     type: 'survey'
   })
+  let scrollToMock
 
   before(function () {
-    wrapper = mount(
-      <FilterStatus
-        task={task}
-      />, {
-        wrappingComponent: Grommet,
-        wrappingComponentProps: { theme: zooTheme }
-      }
-    )
+    scrollToMock = sinon.stub(window, 'scrollTo')
```

Comment:
I think `window.scrollTo` might already be mocked in the global test setup. If not, can you add it, so that it’s available for all tests?

Reply:
You can probably copy the project app test setup. 
https://github.com/zooniverse/front-end-monorepo/blob/feb82afe5f19f269f688c8fb8e3a38fd596016fb/packages/app-project/test/create-global-document.js#L32